<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAKDebugPage</class>
 <widget class="QWidget" name="SAKDebugPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <item row="27" column="1">
    <widget class="QCheckBox" name="bigeEndianCheckBox">
     <property name="text">
      <string>大端顺序</string>
     </property>
    </widget>
   </item>
   <item row="5" column="12" colspan="2">
    <widget class="QCheckBox" name="autoWrapCheckBox">
     <property name="text">
      <string>自动换行</string>
     </property>
    </widget>
   </item>
   <item row="23" column="12" colspan="2">
    <widget class="QPushButton" name="formatSettingsPushButton">
     <property name="text">
      <string>数据分析</string>
     </property>
    </widget>
   </item>
   <item row="20" column="13">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="10" colspan="2">
    <widget class="QCheckBox" name="showTimeCheckBox">
     <property name="text">
      <string>输出时间</string>
     </property>
    </widget>
   </item>
   <item row="13" column="12" colspan="2">
    <widget class="QLabel" name="txSpeedLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="12" colspan="2">
    <widget class="QPushButton" name="outputFilePathPushButton">
     <property name="text">
      <string>输出设置</string>
     </property>
    </widget>
   </item>
   <item row="5" column="10" colspan="2">
    <widget class="QCheckBox" name="showDateCheckBox">
     <property name="text">
      <string>输出日期</string>
     </property>
    </widget>
   </item>
   <item row="9" column="12" colspan="2">
    <widget class="QPushButton" name="saveOutputPushButton">
     <property name="text">
      <string>保存输出</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="19" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">QFrame {
	border:none;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="deviceSettingFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>-------------------------</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="switchPushButton">
        <property name="text">
         <string>打开设备</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="refreshPushButton">
        <property name="text">
         <string>刷新设备</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="12" colspan="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>发送状态</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="21" column="3" colspan="7">
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="27" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="27" column="9">
    <widget class="QPushButton" name="sendPushButton">
     <property name="text">
      <string>发送数据</string>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>数据输入</string>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>输入文本格式</string>
     </property>
    </widget>
   </item>
   <item row="12" column="12" colspan="2">
    <widget class="QLabel" name="rxSpeedLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="26" column="0" colspan="2">
    <widget class="QComboBox" name="crcParameterModelsComboBox"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>数据输出</string>
     </property>
    </widget>
   </item>
   <item row="24" column="10" colspan="2">
    <widget class="QPushButton" name="autoResponseSettingPushButton">
     <property name="text">
      <string>自动回复</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="QComboBox" name="inputModelComboBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="19" column="12" colspan="2">
    <widget class="QPushButton" name="resetRxCountPushButton">
     <property name="text">
      <string>复位接收</string>
     </property>
    </widget>
   </item>
   <item row="27" column="12" colspan="2">
    <widget class="QPushButton" name="moreOhterSettingsPushButton">
     <property name="text">
      <string>更多设置</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>CRC参数模型</string>
     </property>
    </widget>
   </item>
   <item row="17" column="12" colspan="2">
    <widget class="QLabel" name="rxBytesLabel">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="25" column="10" colspan="2">
    <widget class="QPushButton" name="highlightSettingPushButton">
     <property name="text">
      <string>高亮显示</string>
     </property>
    </widget>
   </item>
   <item row="9" column="10" colspan="2">
    <widget class="QPushButton" name="clearOutputPushButton">
     <property name="text">
      <string>清空输出</string>
     </property>
    </widget>
   </item>
   <item row="14" column="12" colspan="2">
    <widget class="QLabel" name="txFramesLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="27" column="3">
    <widget class="QLineEdit" name="cycleTimeLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="11" column="10" colspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="12" colspan="2">
    <widget class="QLabel" name="txLabel">
     <property name="text">
      <string>C&gt;&gt;&gt;&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>设备控制</string>
     </property>
    </widget>
   </item>
   <item row="17" column="10" colspan="2">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>接收字节</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="26" column="10" colspan="2">
    <widget class="QPushButton" name="readWriteSettingPushButton">
     <property name="text">
      <string>读写参数</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QCheckBox" name="addCRCCheckBox">
     <property name="text">
      <string>追加CRC码</string>
     </property>
    </widget>
   </item>
   <item row="27" column="5">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>CRC</string>
     </property>
    </widget>
   </item>
   <item row="23" column="2" rowspan="4" colspan="8">
    <widget class="QTextEdit" name="inputTextEdit"/>
   </item>
   <item row="15" column="10" colspan="2">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>发送字节</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="27" column="8">
    <widget class="QPushButton" name="pushButton_18">
     <property name="text">
      <string>发送预设</string>
     </property>
    </widget>
   </item>
   <item row="23" column="10" colspan="2">
    <widget class="QPushButton" name="transmissionSettingPushButton">
     <property name="text">
      <string>数据转发</string>
     </property>
    </widget>
   </item>
   <item row="24" column="12" colspan="2">
    <widget class="QPushButton" name="pushButton_21">
     <property name="text">
      <string>定时发送</string>
     </property>
    </widget>
   </item>
   <item row="21" column="10" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>其他设置</string>
     </property>
    </widget>
   </item>
   <item row="19" column="10" colspan="2">
    <widget class="QPushButton" name="resetTxCountPushButton">
     <property name="text">
      <string>复位发送</string>
     </property>
    </widget>
   </item>
   <item row="16" column="10" colspan="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>接收帧数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="12" colspan="2">
    <widget class="QComboBox" name="outputModelComboBox"/>
   </item>
   <item row="4" column="10" colspan="2">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>输出文本格式</string>
     </property>
    </widget>
   </item>
   <item row="6" column="12" colspan="2">
    <widget class="QCheckBox" name="showMsCheckBox">
     <property name="text">
      <string>显示毫秒</string>
     </property>
    </widget>
   </item>
   <item row="25" column="12" colspan="2">
    <widget class="QPushButton" name="throughputPushButton">
     <property name="text">
      <string>收发图表</string>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QPushButton" name="saveInputDataPushButton">
     <property name="text">
      <string>保存输入</string>
     </property>
    </widget>
   </item>
   <item row="7" column="10" colspan="2">
    <widget class="QCheckBox" name="showRxDataCheckBox">
     <property name="text">
      <string>显示接收</string>
     </property>
    </widget>
   </item>
   <item row="27" column="7">
    <widget class="QPushButton" name="pushButton_9">
     <property name="text">
      <string>预设数据</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="QPushButton" name="clearInputPushButton">
     <property name="text">
      <string>清空输入</string>
     </property>
    </widget>
   </item>
   <item row="16" column="12" colspan="2">
    <widget class="QLabel" name="rxFramesLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>输入设置</string>
     </property>
    </widget>
   </item>
   <item row="14" column="10" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>发送帧数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="10" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="27" column="2">
    <widget class="QCheckBox" name="cycleEnableCheckBox">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string>wewew</string>
     </property>
     <property name="text">
      <string>定时发送（ms）</string>
     </property>
    </widget>
   </item>
   <item row="27" column="6">
    <widget class="QLabel" name="crcLabel">
     <property name="text">
      <string>0xFFFF</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="20" colspan="8">
    <widget class="QTextBrowser" name="outputTextBroswer"/>
   </item>
   <item row="13" column="10" colspan="2">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>发送速录</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="10" colspan="2">
    <widget class="QCheckBox" name="saveOutputFileToFilecheckBox">
     <property name="text">
      <string>写入文件</string>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>输出设置</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="QPushButton" name="readinFilePushButton">
     <property name="text">
      <string>读入文件</string>
     </property>
    </widget>
   </item>
   <item row="12" column="10" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>接收速率</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="12" colspan="2">
    <widget class="QLabel" name="txBytesLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="12" colspan="2">
    <widget class="QCheckBox" name="showTxDataCheckBox">
     <property name="text">
      <string>显示发送</string>
     </property>
    </widget>
   </item>
   <item row="1" column="10" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>接收状态</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="10" colspan="4">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>数据统计</string>
     </property>
    </widget>
   </item>
   <item row="2" column="10" colspan="2">
    <widget class="QLabel" name="rxLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>C&lt;&lt;&lt;&lt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="10" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
